### DISCUSSION TITLE
Just installed nvim-surround; not working

### DISCUSSION THREAD

**Discussion by k-soomro** [kylechui/nvim-surround#349] [General]
I've just installed this plugin in neovim. I can confirm that it's loaded and I'm using the default config. However, it doesn't seem to actually work for me. Pressing `ysi` goes into insert mode. Am I missing something?

```
NVIM v0.10.1
Build type: Release
LuaJIT 2.1.1720049189
Run "nvim -V1 -v" for more info
```
`nvim-surround` info:
```
version 2.3.0
tag     v2.3.0
branch  main
commit  ec2dc76
```

### Edit

Just to add that I'm on Mac.

**Comment by kylechui** [comment]
Can you send a code snippet for how you set up the function? Calling the `setup` function is necessary for the plugin.

**Reply by k-soomro** [reply]
I copied what's in the instructions. It's an empty function.

**Reply by k-soomro** [reply]
Here is the config. I am using `lazy`:

```lua
 "kylechui/nvim-surround",
version = "*", -- Use for stability; omit to use `main` branch for the latest features
event = "VeryLazy",
config = function()
    require("nvim-surround").setup({
        -- Configuration here, or leave empty to use defaults
    })
end
```

**Comment by k-soomro** [comment]
Ok I just checked and it seems to be working now.
